public/index.html

```diff
@@ -0,0 +1 @@
+View published output <a href="dita/out/index.html">here</a>.
```

Comment:
I wasn't expecting this.  I guess it's not required now that we're hosting from `dita/out`.

Reply:
yes, it is not required, do you want me to remove it?

Reply:
yes please